packages/create-resolve-app/src/create_resolve_app.js

```diff
@@ -63,41 +51,17 @@ const checkAppName = (appName) => {
     return result.validForNewPackages;
 };
 
-const checkIfOnline = () => {
-    return new Promise((resolve) => {
-        dns.lookup('registry.yarnpkg.com', (err) => {
-            if (err != null && process.env.https_proxy) {
-                dns.lookup(url.parse(process.env.https_proxy).hostname, (proxyErr) => {
-                    resolve(proxyErr == null);
-                });
-            } else {
-                resolve(err == null);
-            }
-        });
-    });
-};
-
-const installScripts = (useYarn, scriptsPackage, isOnline) => {
+const installScripts = (scriptsPackage) => {
     return new Promise((resolve, reject) => {
-        let command;
-        let args;
-        if (useYarn) {
-            command = 'yarnpkg';
-            args = ['add', '--exact'];
-            if (!isOnline) {
-                args.push('--offline');
-            }
-
-            if (!isOnline) {
-                log(chalk.yellow('You appear to be offline.'));
-                log(chalk.yellow('Falling back to the local Yarn cache.'));
-                log();
-            }
-        } else {
-            command = 'npm';
-            args = ['install', '--save', '--save-exact', '--loglevel', 'error'];
-        }
-        args.push(scriptsPackage);
+        const command = 'npm';
+        const args = [
+            'install',
+            '--save',
+            '--save-exact',
+            '--loglevel',
+            'error',
+            'resolve-scripts@0.0.26-alpha.4670c701'
```

Comment:
may be scriptsPackage

Reply:
of course... i've just checked that it's work